Spanner/src/Operation.php

```diff
@@ -553,6 +553,7 @@ public function createSnapshot(Session $session, array $res = [], $className = S
      *           labels can be associated with a given session. See
      *           https://goo.gl/xmQnxf for more information on and examples of
      *           labels.
+     *     @type string $creator_role The user created database role which creates the session.
```

Comment:
This is a minor nit, but the convention is to expose options as camel case.

Reply:
This creator_role was present in the generated file. So, we added this in the passing array. I can create a new PR for this, what do you think?